src/shared/components/Pagination/Pagination.tsx

```diff
@@ -16,16 +18,39 @@ export type PaginationProps = {
 export const Pagination: React.FC<PaginationProps> = ({
   itemsPerPage = 0,
   totalCount = 0,
-  maxPaginationLinks = 5,
+  defaultPaginationLength = 5,
```

Comment:
This still works as a maximum right? If it does, I think we should keep the old name or `maxPaginationLength` if you prefer